### ISSUE TITLE
Docker for Cabernet is ignoring the refresh times outline in the scheduled tasks.

### ISSUE THREAD

**Issue by Nostradamus1973** [cabernetwork/cabernet#106]
Good Morning Everyone,

I recently did a fresh install of Kubuntu 23.04 roughly a week ago, and since then, Cabernet isn't obeying the scheduled tasks outlined in the scheduler for refreshing the channels. 
On a side note, the whole not saving the schedules without restarting Cabernet is happening as well with Version: 0.9.13.11 which can be frustrating.

Manually resetting channel data in Data Mgmt does the job properly and things start working until the next expected time interval in scheduled tasks and the same thing happens.

Thank you and have a nice 
[cabernet.log](https://github.com/cabernetwork/cabernet/files/12646891/cabernet.log)
day.

**Comment by rocky4546** [comment]
log says Daddylive channels did refresh and completed at 2023-09-18 06:24:29,037.  XUMO ran correctly at 2023-09-18 06:24:29,037.
No issue found.
The details of why each scheduled task runs is at the debug level if you are interested.

**Comment by Nostradamus1973** [comment]
Hi @rocky4546 , ,thx for getting back to me.

If it only says 6:24 then I think it might prove my point; I had it set to every 30 minutes(to test it) and that log was pulled hours after noticing the problem.
Plus, the inability to save without restarting the docker container is happening as well.

I'll take your advice and set the the log level to Debug and provide another log again in the AM.
Would you suggest any other actions?

**Comment by rocky4546** [comment]
As I said, turning on DEBUG will let you know what is happening with the task.  Running the update channel task every 30 minutes will cause the update channel task to abort, since channel updates rarely occur, so updates every 24 hours minimum are probably appropriate. It's to keep people from banging on the provider and being blocked.

**Comment by Nostradamus1973** [comment]
Thx for clarifying that, I'll take a look at the log and try to see what's happening with the task.

As far as the update channel task, I did plan on putting it back to something in the direction of the default, as I said I was trying to see if it was ignoring the tasks and didn't want to wait hours for it to happen.
I originally changed it because daddylive streams would just drop off and wouldn't load, nothing would work except resetting the channels; after resetting the channels things would return to normal. Today it went 2 hours without needing to be reset and the second time 3 hours, I didn't have time to do this all day so I assumed that I was on to something.

Also, is there a reason why the Scheduled Tasks only save once before having to restart the docker container? 
Thx Rocky, I'll  keep you posted.

**Comment by Nostradamus1973** [comment]
Good Morning Everyone,

So, after a few days of keeping an eye on Cabernet I can say that the Scheduled Tasks ARE working albeit though a broken feature that won't save without restarting the docker container, and I've noticed this issue for the last few revisions, although it can be frustrating to use,  it's workable with a little effort.

For this short experiment, I had to change the Scheduled Tasks to 12 hours because I do daily reboots, so,  I resisted doing Cabernet restarts for an entire day and my viewing experience was absolutely terrible  at times for DaddyLive, hanging and not playing(YET viewable on their DaddyLiveHD.sx site). After restarting, reloading Cabernet everything went back to normal and I could watch DaddyLive. Of course DaddyLive can be down on it's own without Cabernet given they are not mutually exclusive.
I didn't go and change my Scheduled Tasks to hourly or anything aggressive like that after the whole "Banging on the provider" explanation; I'm not into being rude or disrespectful and that explanation made it clear that I shouldn't do something like that.
Until this is resolved/fixed  I'll just continue rebooting Cabernet during those hours on an "As Needed" basis.

Thx and have a great day.
